.travis.yml

```diff
@@ -22,6 +23,7 @@ deploy:
 install:
 - pip install -r requirements.txt
 - pip install pylint circuitpython-build-tools Sphinx sphinx-rtd-theme
+- pip install --force-reinstall pylint==1.9.2
```

Comment:
Just so I'm clear (I think I remember there was some discussion but I may have missed the specifics), are we pinning 1.9.2 for everything at this point?

Reply:
Yes. However, @sommersoft is working on a patch to do it automatically. So, I'm only adding it to the ones that have failing builds based on the new Pylint for now.

We decided that we will update when we're ready to go through the process instead of being forced to do it each time they release a new version and having everything fail suddenly without us being prepared for it.